### ISSUE TITLE
Shortcut catch-all conflict with German keyboard

### ISSUE THREAD

**Issue by petzi53** [rstudio/rstudio#9276]
### System details

    RStudio Edition : Desktop
    RStudio Version : Version 1.4.1106
    OS Version      :  macOS Big Sur 11.3
    R Version       :  4.0.4

### Steps to reproduce the problem

German keyboard: pressing shortcut catch-all ⌘ /

### Describe the problem in detail

Conflicts with shortcuts for numbered lists ⌘ SHIFT 7 (/ is situated above the 7 and produced when the shift key is pressed).

### Describe the behavior you expected
The "insert anything" field/box should appear; instead, it toggles the text between numbered list and the standard paragraph.

- [ x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@petzi53 Thank you for bringing this up! I'll mark this review as we continue development of RStudio and the visual markdown editor.

**Comment by jjallaire** [comment]
Should be addressed with https://github.com/rstudio/rstudio/commit/e5d826ab693a53d1b57171b997e39b7d5a79e08d (removed list keyboard shortcuts as they were already mac-only and have multiple other keyboard gestures available

**Comment by ronblum** [comment]
@jjallaire The behavior is a little bit different now, but I don't think it's the intended behavior. Now
* <kbd>Shift-7</kbd> in the middle of a line inserts `/`
* but <kbd>Shift-7</kbd> without pressing on <kbd>⌘</kbd>at the start of a line results in "Insert Anything"
* and <kbd>⌘-Shift-7</kbd> opens the "Help" menu, which is what <kbd>⌘-Shift-/</kbd> does on a US QWERTY keyboard

**Comment by jjallaire** [comment]
Yes, `Shift-7` at the beginning of a line should show the "Insert Anything" menu (as does the "/" key on English keyboards. If you want to actually type "/" at the beginning of a line just keep typing and the menu will go away. So I _think_ this is operating as intended.

**Comment by ronblum** [comment]
OK—the first two, for <kbd>Shift-7</kbd>, are behaving as intended, then. But:

- On the US keyboard, in which `/` is typed with the <kbd>/<kbd> key
    - <kbd>⌘-/<kbd> (anywhere in the line) results in the "Insert Anything" menu
    - <kbd>⌘-Shift-/<kbd> opens the Help dropdown
- On the German keyboard, in which  `/` is typed with <kbd>Shift-7</kbd>, 
    - <kbd>⌘-Shift-7<kbd> opens the Help dropdown

**Comment by mikebessuille** [comment]
@jjallaire  see Ron's comment

**Comment by ronblum** [comment]
Moving to Spotted Wakerobin to manage the third condition. (Following up from offline discussion.)
